doc/element_where.tex

```diff
@@ -1,4 +1,10 @@
-The \texttt{WHERE} element is used to filter iteration outputs. Results are accepted when the key is equal to the specified \texttt{@value}. The mapping syntax does not specify the data types to be used to evaluate the expression. 
+The \texttt{WHERE} element is used to filter iteration outputs. 
+Results are accepted when the key is equal to the specified \texttt{@value}. 
+\begin{itemize}
+    \item The mapping syntax does not specify the data types to be used to evaluate the expression. 
```

Comment:
Ouch... if we already start with lots of undefinedes, I'm sure we'll get all kinds of funny implementation dependencies, perhaps in WHERE, but in any case with JOIN-s.  Is 3.0==3?  '3'==3? 03==3?  '03'==3? and so on.  I still claim this whole ORM thing is just too difficult and we should drop it at least until we safely can do the simpler things.  But if you insist, at least say "For now, we only define type-preserving comparsions; comparisons requiring type corecions will be defined in a future version of this standard and for now must raise an error" rather than having undefined behaviour right from the start.

Reply:
done as you suggest